pkg/autoops/api/error.go

```diff
@@ -147,6 +147,10 @@ var (
 		codes.FailedPrecondition,
 		"autoops progressive rollout: can not use progressive rollout when feature has prerequisites",
 	)
+	statusProgressiveRolloutFeatureTargetExists = gstatus.New(
+		codes.FailedPrecondition,
+		"autoops progressive rollout: can not use progressive rollout when feature has targets",
```

Comment:
nit: `autoops: can not create a progressive rollout when the feature flag has individual targeting set`